### PR TITLE
Added the newly announced Cloud Practitioner Certification

### PR DESCRIPTION
**Pull request by lobster1234**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -450,6 +450,7 @@ Learning and Career Development
 ### Certifications
 
 -	**Certifications:** AWS offers [**certifications**](https://aws.amazon.com/certification/) for IT professionals who want to demonstrate their knowledge.
+  - [Certified Cloud Practitioner](https://aws.amazon.com/certification/certified-cloud-practitioner/)
```
Comment (MichaelPereira) [inline README.md]: This has no impact on the UI, but you might want to change the indentation between the dash and start of the link to fit with the rest of the list.

Comment (lobster1234) [reply README.md]: Fixed.

**Comment by QuinnyPig** [comment]
No reason this shouldn't have been merged in ages ago. I have failed you, and I'm sorry.

**Comment by lobster1234** [comment]
We're good, @QuinnyPig  :)

**Pull request merged**